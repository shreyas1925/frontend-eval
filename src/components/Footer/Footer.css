.footer{
    margin-top:80px ;
    height: 100px;
    background-color: black;
    width: 100%;
    padding-left: 310px;
    color:white;
    display:flex;
    flex-direction: row;
    align-items: center;
    font-family: Lato, "Helvetica Neue", Helvetica, Arial, sans-serif;
}
.btn{
    height: 50px;
    width: 50px;
    border-radius: 10px;
    margin-left:10px ;
}
.purple-btn{
    background-color: purple;
}
.blue-btn{
    background-color: blue;
}
.gray-btn{
    background-color: gray;
}
.btn-container{
    margin-left: 30px;
}