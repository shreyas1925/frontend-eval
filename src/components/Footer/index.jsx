import * as React from 'react';
import './Footer.css';
export default function Footer() {
  return (
    <div className="footer" data-testid='footer'>
      <h1>
        THEMES
      </h1>
      <div className="btn-container">
        <button type="button" className="btn purple-btn" />
        <button type="button" className="btn blue-btn" />
        <button type="button" className="btn gray-btn" />
      </div>
    </div>
  );
}