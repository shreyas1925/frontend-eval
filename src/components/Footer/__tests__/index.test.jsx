import { render, screen } from "@testing-library/react";
import Footer from "..";
describe("AllButtons", () => {
    it("should render correctly and create a snapshot", () => {
        const {asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
      });

    it("should render correctly", () => {
        render(<Footer />);
        const element = screen.getByTestId("footer");
        expect(element).toBeInTheDocument();
    });
});
