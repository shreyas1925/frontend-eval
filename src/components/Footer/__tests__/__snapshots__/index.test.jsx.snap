// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllButtons should render correctly and create a snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
    data-testid="footer"
  >
    <h1>
      THEMES
    </h1>
    <div
      class="btn-container"
    >
      <button
        class="btn purple-btn"
        type="button"
      />
      <button
        class="btn blue-btn"
        type="button"
      />
      <button
        class="btn gray-btn"
        type="button"
      />
    </div>
  </div>
</DocumentFragment>
`;
